webhook/resources/volume/mutator.go

```diff
@@ -180,7 +184,12 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 		labels[types.LonghornLabelBackupVolume] = bvName
 	}
 
-	patchOp, err := common.GetLonghornLabelsPatchOp(volume, labels)
+	labelsForVolumesFollowsGlobalSettings := datastore.GetLabelsForVolumesFollowsGlobalSettings(volume)
```

Comment:
`SettingNameReplicaAutoBalance` is before 1.4, so do we need to do the upgrade migration?

Reply:
I didn't add an upgrade path for this. This mutator blindly checks the labels during update/creation.
And this labeling makes Longhorn update the volume/engine fields in time when the related settings are modified.

Reply:
I see, it's OK.